homeassistant/components/adax/manifest.json

```
@@ -6,5 +6,6 @@
   "documentation": "https://www.home-assistant.io/integrations/adax",
   "iot_class": "local_polling",
   "loggers": ["adax", "adax_local"],
-  "requirements": ["adax==0.4.0", "Adax-local==0.1.5"]
+  "requirements": ["adax==0.4.0", "Adax-local==0.1.5"],
+  "version": "1.1.0"
```

```suggestion
  "requirements": ["adax==0.4.0", "Adax-local==0.1.5"]
```

Should only be used for custom components